src/WSRequestHandler_Scenes.cpp

```diff
@@ -74,3 +74,71 @@ void WSRequestHandler::HandleGetSceneList(WSRequestHandler* req) {
 
     req->SendOKResponse(data);
 }
+
+/**
+* Changes the order of scene items in the requested scene.
+*
+* @param {String} `scene-name (optional)` Name of the scene to reorder (defaults to current).
```

Comment:
For some reason the documentation parsing is struggling with either the space or the parenthesis in these parameter names. I'll take a look.

I think other optionals have done this instead though
`@param {String (optional)} `scene-name` Name of the scene to reorder (defaults to current).`

Reply:
Sounds good i can change that over